app/lib/submission_builder/documents/adv_ctc_irs1040.rb

```diff
@@ -14,38 +14,34 @@ def dependent_xml(xml, dependent)
         xml.DependentDetail do
           xml.DependentFirstNm dependent.first_name
           xml.DependentLastNm dependent.last_name
-          xml.DependentNameControlTxt person_name_control_type(dependent.last_name)
+          xml.DependentNameControlTxt person_name_control_type(dependent.first_name)
           xml.DependentSSN dependent.ssn
           xml.DependentRelationshipCd dependent.relationship&.upcase
-          xml.EligibleForChildTaxCreditInd "X" if dependent.eligible_for_child_tax_credit?(2021)
```

Comment:
I was originally confused about the year we were supposed to be comparing to for this determination.